<launch>
  <arg name="global_frame" default="world"/>
  <arg name="sensor_frame" default="sensor_frame"/>

  <node name="simple_collector_node" pkg="fkie_measurement_server" exec="measurement_collector_node.py">
    <param name="utm_zone_number" value="32"/>
    <param name="utm_zone_letter" value="U"/>
    <param name="topic_measurement" value="measurement"/>
    <param name="global_frame" value="$(var global_frame)"/>
  </node>

  <node name="tf_world_sensor_frame" pkg="tf2_ros" exec="static_transform_publisher" args="1.0 0.5 0.6 0 0 0 $(var global_frame) $(var sensor_frame)">
  </node>

</launch>